app/controllers/application.js

```diff
@@ -0,0 +1,41 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+	isExpanded: true,
+	isNotLogin: Ember.computed('this.currentPath', function() {
+	  	if (this.currentPath !== "login") {
+	  		return true
+	  	} else {
+	  		return false
+	  	}
+	}),
+	isExperiments: Ember.computed('this.currentPath', function() {
+	  	if (this.currentPath === "experiments.index") {
+	  		return true
+	  	} else {
+	  		return false
+	  	}
+	}),
+	isParticipants: Ember.computed('this.currentPath', function() {
+	  	if (this.currentPath === "participants") {
+	  		return true
+	  	} else {
+	  		return false
+	  	}
+	}),
+	sizeContainer: function() {
```

Comment:
Actually this is a bit more complicated than it seems. Using the toggleable `isExpanded` is a good approach. 

I really don't like polling the window size, but this could just be superstition on my part and may be a reliable method. There's a CSS-class based implementation here: https://github.com/BlackrockDigital/startbootstrap-simple-sidebar that might be cleaner.